example/use_cases/apartment_application/apartment_application.go

```diff
@@ -0,0 +1,150 @@
+// This is a full example flow of an apartment verifying the age of a potential tenant.
+
+// The apartment will create a Presentation Request that is to be fulfilled by the tenant.
+// The tenant will fulfil the Presentation Request by submitting a Presentation Submission.
+// This presentation submission will contain a verifiable credential that has been previously issued and signed from the government issuer.
+
+// The tenant will verify that the apartment's presentation request is valid and the apartment will also verify that the tenant's
+// presentation submission is valid.
+
+// At the end the apartment will verify the authenticity of the presentation submission and will be able to verify the birthdate of the tenant.
+
+package main
+
+import (
+	"crypto/ed25519"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/goccy/go-json"
+	"github.com/stretchr/testify/assert"
+)
+
+func main() {
+
+	/**
+		 Step 1: Create new entities as DIDs. Govt Issuer, User Holder, and Apartment Verifier
+	**/
+
+	// User Holder
+	holderDIDPrivateKey, holderDIDKey, _ := did.GenerateDIDKey(crypto.Ed25519)
```

Comment:
the other examples do some light error handling like this, what do you think? https://github.com/TBD54566975/ssi-sdk/blob/main/example/vc/vc.go#L54